src/main/java/cinema/model/Movie.java

```diff
@@ -5,8 +5,10 @@
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
+import javax.persistence.Table;
 
 @Entity
+@Table(name = "movie")
```

Comment:
If you specify table names, make it plural

Reply:
Fixed